apps/aehttpserver/lib/aehttpserver/web/channels/notify.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Aehttpserver.Web.Notify do
+  alias Aeutil.Serialization
+
+  def broadcast_new_transaction_in_the_pool(tx) do
+    Aehttpserver.Web.Endpoint.broadcast!("room:notifications","new_tx:" <> Base.encode16(tx.data.from_acc), %{"body" => Serialization.tx(tx, :serialize)})
+    Aehttpserver.Web.Endpoint.broadcast!("room:notifications","new_tx:" <> Base.encode16(tx.data.to_acc), %{"body" => Serialization.tx(tx, :serialize)})
+    Aehttpserver.Web.Endpoint.broadcast!("room:notifications", "new_transaction_in_the_pool", %{"body" => Serialization.tx(tx, :serialize)})
+  end
+
+  def broadcast_new_block_added_to_chain_and_new_mined_tx(block) do
+    Enum.each(block.txs, fn(tx) -> 
+      Aehttpserver.Web.Endpoint.broadcast!("room:notifications", "new_mined_tx_everyone", %{"body" => Serialization.tx(tx, :serialize)})
+      if tx.data.from_acc != nil and tx.data.to_acc != nil do
+        Aehttpserver.Web.Endpoint.broadcast!("room:notifications", "new_mined_tx:" <> Base.encode16(tx.data.from_acc), %{"body" => Serialization.tx(tx, :serialize)})
```

Comment:
I think we should also inform the frontend about block height and hash.

Reply:
height is included in the block header

Reply:
But the block isn't send only the tx is.

Reply:
the block is sent as well https://github.com/aeternity/elixir-research/pull/189/files#diff-96c1edf72c3ba21f39b23ced455d427cR18